README.md

```
@@ -1 +1,8 @@
-# Peakle-Shuttle-Web-Back
\ No newline at end of file
+# Peakle-Shuttle-Web-Back
+![CodeRabbit Pull Request Reviews](https://img.shields.io/coderabbit/prs/github/Peakle-Shuttle/Peakle-Shuttle-Web-Back?utm_source=oss&utm_medium=github&utm_campaign=Peakle-Shuttle%2FPeakle-Shuttle-Web-Back&labelColor=171717&color=FF570A&link=https%3A%2F%2Fcoderabbit.ai&label=CodeRabbit+Reviews)
+
+| `./gradlew bootRun --args='--spring.profiles.active=local'` | 애플리케이션 실행 |
+Swagger UI: http://localhost:8080/swagger-ui.html
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)